roles/metrics/README.md

```diff
@@ -0,0 +1,25 @@
+theforeman.operations.metrics
+=============================
+
+Setup Performace Co-Pilot and Grafana to grather metrics of a Foreman installation.
+
+This role is heavily based on the [performancecopilot.metrics](https://github.com/performancecopilot/ansible-pcp) collection.
+
+Role Variables
+--------------
+
+* `foreman_metrics_url`: The URL of the OpenMetrics endpoint of Foreman, defaults to `https://{{ ansible_fqdn | default('localhost') }}/metrics`.
+* `foreman_pcp_optional_agents`: The optional PCP agents to enable, defaults to `[apache, openmetrics, postgresql, redis]`.
+* `foreman_metrics_grafana_enabled`: Whether or not Grafana should be installed and configured on the system, defaults to `true` on Red Hat family systems, and to `false` otherwise.
+* `foreman_metrics_grafana_pmproxy_url`: The URL of the `pmproxy` service to be used as the Grafana datasource, defaults to `http://{{ ansible_fqdn | default('localhost') }}:44322`.
```

Comment:
Does this port need to be mentioned in our ports guide with the addition of our monitoring guide?

Reply:
Do we mention the Candlepin 23443 port? It's a local connection (unless you do remote-grafana, but then you'd usually also have the pmproxy remotely), so not really needed.

Reply:
We don't, but we specifically set Candlepin to localhost. This appears that it would set it to the hostname (so I assume the firewall would not incorporate this as a in and out connection). Will pmproxy always be remote if Grafana is remote?